<html>
<head>
<title>TestNG:  Test</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>Test</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>4/2/0</td>
</tr><tr>
<td>Started on:</td><td>Thu Mar 09 16:44:56 IST 2023</td>
</tr>
<tr><td>Total time:</td><td>115 seconds (115938 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='testcases.logintestcase.login()'><b>login</b><br>Test class: testcases.logintestcase<br>Parameters: vishal101@gmail.com, vi@123</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	error message not matching expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 .] but found [New Customer
Register Account
By creating an account you will be able to shop faster, be up to date on an order&apos;s status, and keep track of the orders you have previously made.
Continue
Returning Customer
I am a returning customer
E-Mail Address
Password
Forgotten Password
Login
Register
Forgotten Password
My Account
Address Book
Wish List
Order History
Downloads
Recurring payments
Reward Points
Returns
Transactions
Newsletter]
	at Resources.commonmethods.handleassertion(commonmethods.java:15)
	at testcases.logintestcase.login(logintestcase.java:53)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace77738832", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace77738832'><pre>java.lang.AssertionError: The following asserts failed:
	error message not matching expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 .] but found [New Customer
Register Account
By creating an account you will be able to shop faster, be up to date on an order&apos;s status, and keep track of the orders you have previously made.
Continue
Returning Customer
I am a returning customer
E-Mail Address
Password
Forgotten Password
Login
Register
Forgotten Password
My Account
Address Book
Wish List
Order History
Downloads
Recurring payments
Reward Points
Returns
Transactions
Newsletter]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
	at Resources.commonmethods.handleassertion(commonmethods.java:15)
	at testcases.logintestcase.login(logintestcase.java:53)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:794)
	at org.testng.TestRunner.run(TestRunner.java:596)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at org.testng.TestNG.runSuites(TestNG.java:1063)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>18</td>
<td>testcases.logintestcase@79924b</td></tr>
<tr>
<td title='testcases.logintestcase.login()'><b>login</b><br>Test class: testcases.logintestcase<br>Parameters: abhi123@gmail.com, abhi456</td>
<td><div><pre>java.lang.AssertionError: The following asserts failed:
	error message not matching expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 .] but found [New Customer
Register Account
By creating an account you will be able to shop faster, be up to date on an order&apos;s status, and keep track of the orders you have previously made.
Continue
Returning Customer
I am a returning customer
E-Mail Address
Password
Forgotten Password
Login
Register
Forgotten Password
My Account
Address Book
Wish List
Order History
Downloads
Recurring payments
Reward Points
Returns
Transactions
Newsletter]
	at Resources.commonmethods.handleassertion(commonmethods.java:15)
	at testcases.logintestcase.login(logintestcase.java:53)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1250816994", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1250816994'><pre>java.lang.AssertionError: The following asserts failed:
	error message not matching expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 .] but found [New Customer
Register Account
By creating an account you will be able to shop faster, be up to date on an order&apos;s status, and keep track of the orders you have previously made.
Continue
Returning Customer
I am a returning customer
E-Mail Address
Password
Forgotten Password
Login
Register
Forgotten Password
My Account
Address Book
Wish List
Order History
Downloads
Recurring payments
Reward Points
Returns
Transactions
Newsletter]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
	at Resources.commonmethods.handleassertion(commonmethods.java:15)
	at testcases.logintestcase.login(logintestcase.java:53)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:794)
	at org.testng.TestRunner.run(TestRunner.java:596)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at org.testng.TestNG.runSuites(TestNG.java:1063)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>18</td>
<td>testcases.logintestcase@79924b</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='testcases.verifyheader.header()'><b>header</b><br>Test class: testcases.verifyheader</td>
<td></td>
<td>7</td>
<td>testcases.verifyheader@6eda5c9</td></tr>
<tr>
<td title='testcases.signuptestcase.login()'><b>login</b><br>Test class: testcases.signuptestcase</td>
<td></td>
<td>5</td>
<td>testcases.signuptestcase@7b9a4292</td></tr>
<tr>
<td title='testcases.verifyfootertests.footer()'><b>footer</b><br>Test class: testcases.verifyfootertests</td>
<td></td>
<td>0</td>
<td>testcases.verifyfootertests@4fb0f2b9</td></tr>
<tr>
<td title='testcases.verifymenubarandaddcart.menu()'><b>menu</b><br>Test class: testcases.verifymenubarandaddcart</td>
<td></td>
<td>15</td>
<td>testcases.verifymenubarandaddcart@4a94ee4</td></tr>
</table><p>
</body>
</html>